src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationevents/models/enums/IntegrationEventTypes.kt

```diff
@@ -54,22 +56,19 @@ object RegisterTypes {
   const val WARRANT_SUMMONS_CODE = "WRSM" // Outstanding warrant or summons
 }
 
-enum class EventTypes(val integrationEventTypes: IntegrationEventTypes, val messageFilterAttributes: List<String>, val path: String) {
-  DYNAMIC_RISKS(IntegrationEventTypes.DYNAMIC_RISKS_CHANGED, DYNAMIC_RISKS_REGISTER_TYPES, "risks/dynamic"),
-  PROBATION_STATUS(IntegrationEventTypes.PROBATION_STATUS_CHANGED, PROBATION_STATUS_REGISTER_TYPES, "status-information"),
-  MAPPA_DETAIL(IntegrationEventTypes.MAPPA_DETAIL_CHANGED, MAPPA_DETAIL_REGISTER_TYPES, "risks/mappadetail"),
-  RISK_SCORE(IntegrationEventTypes.RISK_SCORE_CHANGED, RISK_SCORE_TYPES, "risks/scores"),
-  KEY_DATES_PRISONER_RELEASE(IntegrationEventTypes.KEY_DATES_AND_ADJUSTMENTS_PRISONER_RELEASE, PRISONER_RELEASE_TYPES, "sentences/latest-key-dates-and-adjustments"),
+enum class EventTypes(val integrationEventTypes: IntegrationEventTypes, val path: String) {
+  DYNAMIC_RISKS(IntegrationEventTypes.DYNAMIC_RISKS_CHANGED, "/risks/dynamic"),
+  PROBATION_STATUS(IntegrationEventTypes.PROBATION_STATUS_CHANGED, "/status-information"),
+  MAPPA_DETAIL(IntegrationEventTypes.MAPPA_DETAIL_CHANGED, "/risks/mappadetail"),
+  RISK_SCORE(IntegrationEventTypes.RISK_SCORE_CHANGED, "/risks/scores"),
+  KEY_DATES_PRISONER_RELEASE(IntegrationEventTypes.KEY_DATES_AND_ADJUSTMENTS_PRISONER_RELEASE, "/sentences/latest-key-dates-and-adjustments"),
+  PERSON_STATUS(IntegrationEventTypes.PERSON_STATUS_CHANGED, ""),
   ;
 
   companion object {
     fun from(eventType: IntegrationEventTypes, message: HmppsDomainEventMessage): EventTypes? =
       EventTypes.entries.firstOrNull {
-        it.integrationEventTypes == eventType &&
```

Comment:
Do we no longer filter on registration types etc?  Or is that happening somewhere else now?

Reply:
it is happening elsewhere:

  companion object {
    fun from(value: String, registerType: String?): IntegrationEventTypes? =
      IntegrationEventTypes.entries.firstOrNull {
        when (registerType) {
          null -> it.upstreamEventTypes.contains(value)
          else -> it.upstreamEventTypes.contains(value) && it.registerTypes!!.contains(registerType)
        }
      }
  }